common/v2/components/AccountList.tsx

```diff
@@ -30,9 +30,34 @@ import { TUuid } from 'v2/types/uuid';
 const Label = styled.span`
   display: flex;
   align-items: center;
-  @media (min-width: ${BREAK_POINTS.SCREEN_SM} {
+  @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
+    font-weight: bold;
+  }
+`;
+
+const LabelWithWallet = styled.span`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
     font-weight: bold;
-  })
+  }
+`;
+
+const WalletTypeLabelContainer = styled.div`
+  display: block;
+`;
+
+const WalletTypeLabel = styled.p`
+  background: #b7bfc6;
+  display: inline-block;
+  text-align: center;
+  border-radius: 600px;
+  font-size: 0.8rem;
+  padding: 3px 6px;
+  color: ${COLORS.WHITE};
+  @media (min-width: ${BREAK_POINTS.SCREEN_SM}) {
+    font-size: 0.6rem;
+  }
```

Comment:
There is a font jump when changing screen size.
IMHO, you can set the font-size once using `em`.